### PR TITLE
feat: add spec for `onboardbase`

### PR DESCRIPTION
**Pull request by iamnasirudeen**
Added Specs for Onboardbase

### CODE REVIEW

File: src/onboardbase.ts [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+    {
+      name: "build",
+      description: "Run a command with secrets injected into the environment",
+      options: [
+        { name: ["-h", "--help"], description: "show CLI help" },
+        { name: ["-p", "--project"], description: "name of project", args: {} },
+        {
+          name: ["-e", "--environment"],
+          description: "Environment to run by default",
+          args: {},
```
Comment (fedeci) [inline src/onboardbase.ts:15]:
```suggestion
          args: {
            name: "environment",
          },
```

File: src/onboardbase.ts [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+const completionSpec: Fig.Spec = {
+  name: "onboardbase",
+  subcommands: [
+    {
+      name: "build",
+      description: "Run a command with secrets injected into the environment",
+      options: [
+        { name: ["-h", "--help"], description: "show CLI help" },
+        { name: ["-p", "--project"], description: "name of project", args: {} },
```
Comment (fedeci) [inline src/onboardbase.ts:11]:
```suggestion
        { name: ["-p", "--project"], description: "name of project", args: { name: "project name" } },
```

File: src/onboardbase.ts [hunk 3]
```diff
@@ -0,0 +1,191 @@
+// Autogenerated by @withfig/oclif
+
+const completionSpec: Fig.Spec = {
+  name: "onboardbase",
```
Comment (fedeci) [inline src/onboardbase.ts:4]:
```suggestion
  name: "onboardbase",
  description:  "Onboardbase manages app secrets and credentials securely with team work and contractors in mind",
```

**Review by fedeci — CHANGES_REQUESTED** [review]
Thank you this spec! Could you please run `npm run lint:fix` and add basic names to args? Then we are ready to merge this 🚀

**Comment by fedeci** [comment]
> Thank you this spec! Could you please run `npm run lint:fix` and add basic names to args? Then we are ready to merge this 🚀

There are still a lot of args missing the name :) We need it to let the user know what to insert, so it is quite important.

**Review by iamnasirudeen — COMMENTED** [review]
Update the args

**Review by fedeci — APPROVED** [review]
Thanks!

**Pull request merged**